application-analytics-ui/src/main/resources/Analytics/Code/Macros/BrowserEngines.xml

```diff
@@ -292,7 +292,11 @@
 #end
 #if ($services.licensing.licensor.hasLicensureForEntity(
   $services.model.createDocumentReference('', ['Analytics', 'Code'], 'ConfigurationClass')))
-  #executeMacro()
+  #if ($xwiki.hasAccessLevel('view', 'Analytics.WebHome'))
```

Comment:
Shouldn't these also check for edit right?

Reply:
I think the best way to handle this is to check for viewing rights because the admin may want to grant someone viewing rights on the macros without the ability to edit them.